strawberry/experimental/pydantic/_compat.py

```diff
@@ -17,6 +17,9 @@
 IS_PYDANTIC_V2: bool = PYDANTIC_VERSION.startswith("2.")
 IS_PYDANTIC_V1: bool = not IS_PYDANTIC_V2
 
+if IS_PYDANTIC_V2:
```

Comment:
I'm not sure this is absolutely necessary, but when I was running the tests locally with pydantic v2 all of the mypy automated tests were failing without this import.